feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreen.kt

```diff
@@ -257,7 +257,7 @@ fun PreviewTimetableScreenDark() {
                     ),
                 ),
                 TimetableUiType.Grid,
-                null,
+                false,
```

Comment:
This is an existing implementation, but it is difficult to review because I can only see a change from null to false. Could you add a named parameter for this? 🙇

Reply:
I add parameter names to this preview :smiley: 
(https://github.com/DroidKaigi/conference-app-2023/pull/1131/commits/66f1cce05a15f6a39a2049ca2096fec77048a286)